src/libsystemd-network/dhcp-server-internal.h

```diff
@@ -63,6 +63,8 @@ struct sd_dhcp_server {
         struct in_addr *ntp, *dns;
         unsigned n_ntp, n_dns;
 
+        uint8_t emit_router;
```

Comment:
Because this is a struct and all other members of that struct are also defined as strictly-sized datatypes. I don't like bool in structs, where the offset is important. But I can change it if you prefer to have a bool in struct sd_dhcp_server.